README.md

```diff
@@ -6,38 +6,54 @@
 
 ### About
 
-This driver allows Kubernetes to access Azure Lustre filesystem.
+This driver allows Kubernetes to access Azure Lustre file system.
 
-#### csi plugin name: `azurelustre.csi.azure.com`
+- CSI plugin name: `azurelustre.csi.azure.com`
+- Project status: under early development
 
-### Project status: under early development
+&nbsp;
 
 ### Container Images & Kubernetes Compatibility:
 
-|driver version  |Image                                             | supported k8s version |
-|----------------|--------------------------------------------------|-----------------------|
-|main branch     |mcr.microsoft.com/k8s/csi/azurelustre-csi:latest  | 1.21+                 |
+| Driver version  | Image                                                         | Supported k8s version | Lustre client version |
+|-----------------|---------------------------------------------------------------|-----------------------|-----------------------|
+| main branch     | mcr.microsoft.com/oss/kubernetes-csi/azurelustre-csi:latest   | 1.21+                 | 2.14                  |
+| v0.1.1          | mcr.microsoft.com/oss/kubernetes-csi/azurelustre-csi:v.0.1.1  | 1.21+                 | 2.14                  |
```

Comment:
typo: `v.0.1.1`

Reply:
Thanks Andy! I will fix it in 0.1.3 release